Products/PleiadesEntity/browser/adapters/__init__.py

```diff
@@ -389,6 +392,10 @@ def end(self):
 
     @export_config(json=False)
     def extent(self):
+        fullGeometry = self.context.getGeometryJSON()
```

Comment:
I think this may belong in `Products.PleiadesEntity.browser.adapters.Location.LocationExportAdapter.geometry` rather than this super class, unless there are other Place sub-objects that need to handle full geometries this way.

@paregorios is there any reason to retain the convex hull in the Location exports, or should we always only include the full geometry? Perhaps we need multiple geometry attributes in the export (`geometry`, `hull`, `bbox`)?

If we're only interested in the complete geometry, and never want the convex hull polygon, then maybe we should be storing the full geometry in the catalog metadata `zgeo_geometry` instead of the convex hull generated by `IExtent`. This may not be worth the effort, since we seem to always have the full object and not just the brain, and doing this would require reindexing a bunch of stuff.